research/lambda-papers/index.html

```diff
@@ -1,31 +1,173 @@
-<h1>The Original 'Lambda Papers' by Guy Steele and Gerald Sussman</h1>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>The Lambda Papers</title>
+  <link rel="stylesheet" href="/schemeorg.css">
+  <link rel="stylesheet" href="../../schemeorg.css">
```

Comment:
Same here and everywhere else it's referenced twice.

Would you please move the ".css" file to one place and make sure that it's referenced there from everywhere?